pkg/certificate/authorization/service.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package authorization
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// ErrService is the default error class for the authorization service.
+var ErrService = errs.Class("authorization service error")
+
+// Service is the authorization service.
+type Service struct {
+	log *zap.Logger
+	db  *DB
+}
+
+// NewService creates a new authorization service.
+func NewService(log *zap.Logger, db *DB) *Service {
+	return &Service{
+		log: log,
+		db:  db,
+	}
+}
+
+// GetOrCreate will return an authorization for the given user ID.
+func (service *Service) GetOrCreate(ctx context.Context, userID string) (_ *Token, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	if userID == "" {
+		msg := "error getting authorizations"
+		err = ErrService.Wrap(err)
+		service.log.Error(msg, zap.Error(err))
```

Comment:
There is no need to log this error is a user input error.

Reply:
sorry this keeps happening.. it's a copy/paste/neglect issue on my part.

Reply:
This has been forgotten.

Reply:
Ah, thanks... i lost track of it.